spec/string_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+RSpec.describe String do
+
+  describe '#uncapitalize' do
+    it 'downcases the first letter of self, returning a new string' do
+      expect('Tomato'.uncapitalize).to eq('tomato')
```

Comment:
Can you modify the spec to have some other capitalized letters in the word to ensure that it is only the first letter that is being downcased?

Reply:
done!